core/lib/spree/core.rb

```diff
@@ -91,3 +91,6 @@ class DestroyWithOrdersError < StandardError; end
 require 'spree/core/controller_helpers/strong_parameters'
 require 'spree/core/unreturned_item_charger'
 require 'spree/core/role_configuration'
+
+require 'spree/mailer_previews/order_preview'
+require 'spree/mailer_previews/carton_preview'
```

Comment:
Maybe iterate over the mailer_previews here. This way when we add or remove over time, we won't have to edit this file.